src/main/java/com/synopsys/integration/alert/provider/blackduck/filter/BlackDuckProjectNameExtractor.java

```diff
@@ -81,7 +81,7 @@ public BlackDuckProjectNameExtractor(BlackDuckProperties blackDuckProperties) {
             BomEditNotificationView notificationView = cache.getTypedContent(notification, BomEditNotificationView.class);
             String bomComponentUri = notificationView.getContent().getBomComponent();
             try {
-                VersionBomComponentView bomComponentView = blackDuckService.getResponse(bomComponentUri, VersionBomComponentView.class);
```

Comment:
VersionBomComponentView is deprecated and is replaced by ProjectVersionComponentView